### PR TITLE
Testing code against Redis 7

### PR DESCRIPTION
**Pull request by zaherg**
This PR make sure that tests will pass against Redis 7.

PS: this PR will just make tests pass against redis 7, it does not add any new feature or changes which was implemented in redis 7 like the changes in the replication naming.

### CODE REVIEW

File: tests/Predis/Command/Redis/CONFIG_Test.php [hunk 1]
```diff
@@ -143,7 +143,7 @@ public function testReturnsTrueOnSuccessfulConfiguration(): void
     public function testThrowsExceptionWhenSettingUnknownConfiguration(): void
     {
         $this->expectException('Predis\Response\ServerException');
-        $this->expectExceptionMessage('ERR Unsupported CONFIG parameter: foo');
+        $this->expectExceptionMessageMatches("/ERR|CONFIG|foo|arguments|parameter/");
```
Comment (tillkruss) [inline tests/Predis/Command/Redis/CONFIG_Test.php:146]: Should we maybe have two full sentences here? That will make it easier to read.

**Review by tillkruss — CHANGES_REQUESTED** [review]

File: tests/Predis/Command/Redis/CONFIG_Test.php (hunk 1 above)
Comment (zaherg) [reply tests/Predis/Command/Redis/CONFIG_Test.php:146]: The sentence is different between 7 and anything lower so instead I was aiming to check that the key words are there which will work on all versions.

If you prefer to have a full sentence, I can do a small version check and test its content based on redis version

Comment (tillkruss) [reply tests/Predis/Command/Redis/CONFIG_Test.php:146]: Yeah, I think `(old error message|new error msg)` will be less difficult to read.

Comment (tillkruss) [reply tests/Predis/Command/Redis/CONFIG_Test.php]: Even better!

**Pull request merged**